### PR TITLE
Adding the MD files for the custom widgets TagCloud and MultipleDropDown

### PR DESCRIPTION
**Pull request by Sathishkanniyappan**
## Description
Need to add md file for the custom widgets(TagCloud and MultipleDropDown)
                
## Solution description
We have added the md file for the above mentioned custom widgets and added the respective images.

### CODE REVIEW

File: multipledropdown/doc/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+**MultipleDropDown**
+
+MultipleDropDown custom widget is used to visualize the data as in the below image.
```
Comment (balajeee22) [inline multipledropdown/doc/ReadMe.md:3]: Have proper content. we have already prepared the use cases. get the proper content. this looks useless.

File: tagcloud/doc/ReadMe.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+**TagCloud**
+
+This is TagCloud to visualize the data as in the below image.
```
Comment (balajeee22) [inline tagcloud/doc/ReadMe.md:3]: Have proper content. we have already prepared the use cases. get the proper content. this looks useless.

Comment (Sathishkanniyappan) [reply tagcloud/doc/ReadMe.md]: Corrected the MD file content as per the suggestion.

File: multipledropdown/doc/ReadMe.md (hunk 1 above)
Comment (Sathishkanniyappan) [reply multipledropdown/doc/ReadMe.md]: Corrected the MD file content as per the suggestion.

**Review by balajeee22 — APPROVED** [review]

**Pull request merged**